src/main/java/org/broadinstitute/barclay/utils/Utils.java

```diff
@@ -136,4 +141,31 @@ public static boolean equals(Object lhs, Object rhs) {
         return lhs == null && rhs == null || lhs != null && lhs.equals(rhs);
     }
 
+    /**
+     * a utility method to wrap multiple lines of text, while respecting the original
+     * newlines.
+     *
+     * @param input the String of text to wrap
+     * @param width the width in characters into which to wrap the text
+     * @return the original string with newlines added, and starting spaces trimmed
+     * so that the resulting lines fit inside a column of with characters.
+     */
+    public static String wrapParagraph(final String input, final int width)  {
```

Comment:
Non-null check for `input` (if not, NPE will be thrown in line 156).

Reply:
done.